desktop_version/src/Render.cpp

```diff
@@ -2040,6 +2040,143 @@ static void draw_roomname_menu(void)
  * the same in Flip Mode. */
 #define FLIP(y, h) (graphics.flipmode ? 220 - (y) - (h) : (y))
 
+void rendermap(void)
+{
+#ifndef NO_CUSTOM_LEVELS
+    if (map.custommode)
+    {
+        //draw the map image
+        graphics.drawpixeltextbox(35 + map.custommmxoff, 16 + map.custommmyoff, map.custommmxsize + 10, map.custommmysize + 10, 65, 185, 207);
+        graphics.drawpartimage(graphics.minimap_mounted ? 1 : 12, 40 + map.custommmxoff, 21 + map.custommmyoff, map.custommmxsize, map.custommmysize);
+        return;
+     }
+#endif /* NO_CUSTOM_LEVELS */
+
+    //draw the map image
+    graphics.drawpixeltextbox(35, 16, 250, 190, 65, 185, 207);
+    graphics.drawimage(1, 40, 21, false);
+}
+
+void rendermapfog(void)
+{
+    int mapwidth = map.custommode ? map.customwidth : 20;
+    int mapheight = map.custommode ? map.customheight : 20;
+    int mapzoom = map.custommode ? map.customzoom : 1;
+    int mapxoff = map.custommode ? map.custommmxoff : 0;
+    int mapyoff = map.custommode ? map.custommmyoff : 0;
+
+    // Draw the fog of war
```

Comment:
The comment is not needed when this routine is in a separate function and it's in the name of the function.
```suggestion
```